common/http.go

```diff
@@ -0,0 +1,24 @@
+package common
+
+import (
+	"net/http"
+)
+
+type logHandler struct {
+	handler http.Handler
+}
+
+// LogHandler creates a http.Handler for logging HTTP requests. The handler
+// wraps a given handler h.
+//
+// A format of the logging is the following:
+// $METHOD $REQUEST_URI
+func LogHandler(h http.Handler) http.Handler {
+	return &logHandler{handler: h}
+}
+
+// ServeHTTP implements the http.Handler interface.
+func (l *logHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	Log.Debugf("%s %s", r.Method, r.URL.RequestURI())
+	l.handler.ServeHTTP(w, r)
+}
```

Comment:
I'd call this thing `LoggingHTTPHandler` and dispense with *all* the comments above.